doc/man3/RSA_set_method.pod

```diff
@@ -168,6 +169,11 @@ not currently exist).
 
 L<RSA_new(3)>
 
+=head1 HISTORY
+
+The RSA_null_method(), which was a partial attempt to avoid patent issues,
+was replaced to always return NULL in OpenSSL 1.1.0d.
```

Comment:
Make this 1.1.1. I'd rather not backport this one commit (the other commits are fine for backport).